runtime/doc/news.txt

```diff
@@ -63,7 +63,7 @@ The following new APIs or features were added.
 <
   (or the Vimscript equivalent) to their |config| file.
 
-• Run Lua scripts from your shell using |-l|. >
+• Added support for running Lua scripts from shell using |-l|. >
```

Comment:
This is not an improvement, and it's not clear yet what kind of style news.txt will have, so "consistency" is premature here. 

This kind of change also generally should be collected in a "typos" PR if it's needed at all.